src/gov-action-contracts/address-registries/L1TokenBridgeAddressRegistry.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "./interfaces.sol";
+
+contract L1TokenBridgeAddressRegistry is ITokenBridgeAddressRegistry {
```

Comment:
We should probably combine this with the L1AddressReg, so that if an action contract needs addresses from both we wont have to provide two address registries